tests/05_message.ts

```diff
@@ -29,3 +39,50 @@ Deno.test({
     assertEquals(message.embeds[0].title, "Discordeno Test");
   },
 });
+
+Deno.test({
+  name: "[message] pin a message in a channel",
+  fn() {
+    pin(tempData.channelID, tempData.messageID);
```

Comment:
We should also test this through getters imo.
`message.pin(channelID, anotherMessge.id)`

Reply:
getters should be tested for everything as well

Reply:
@Skillz4Killz, the getter is basically calling `pin()` internally.